Update WebRTC build

dbrgn opened the pull request:
New version is at commit 1a35fbd9c38cbf2f454b4808d28e2ca0e93853cf (current master).

https://chromium.googlesource.com/external/webrtc/+/1a35fbd9c38cbf2f454b4808d28e2ca0e93853cf

lgrahl reviewed (commented):
Okay. I guess we need to test it manually?

dbrgn commented:
Yes

threema-danilo committed:
Add helper script for updating WebRTC builds

threema-danilo committed:
Update WebRTC build: Release 71

New version is at commit d2362beb1744676b11942832c44c7925911b14ec

https://chromium.googlesource.com/external/webrtc/+log/branch-heads/71

dbrgn commented:
Updated to Release 71 (https://groups.google.com/forum/#!topic/discuss-webrtc/HUpIxlDlkSE).

Tested briefly. Works fine so far. (Requires some changes on the user side.)
